tests/Test/Dscp/DB/DSL/Interpret/SimpleTxDB.hs

```diff
@@ -48,27 +48,27 @@ mkLinAlgPrivateTx :: StudentKeySeed -> EducatorKeySeed -> PrivateTx
 mkLinAlgPrivateTx sKeySeed eKeySeed =
     let studentKey = mkPubKey sKeySeed
         educatorKeyPair = mkKeyPair eKeySeed
-    in mkPrivateTx courseLinearAlg C studentKey educatorKeyPair
+    in mkPrivateTx courseLinearAlg 60 studentKey educatorKeyPair
```

Comment:
Someone is going to suffer when grade representation changes again. How about extracting several constants?

Reply:
Fair enough, I just needed to fix tests then.